ISISComputingGroup/EPICS-ioc
# Ticket 4488: Removed attocube as not used

### Description of work

See https://github.com/ISISComputingGroup/IBEX/issues/4488

### To test

* Confirm that a `make` from the ioc directory still works.
* Confirm there is no trace of the attocube

---

#### Code Review

    - [PV naming](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming).
    - [Disable records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Disable-records)
    - [Record simulation](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Record-Simulation)
    - [Finishing touches](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Finishing-Touches)

### Functional Tests

- IOC responds correctly in:

### Final steps
